lib/output.ts

```diff
@@ -13,18 +13,31 @@ export function mismatchingVersionsToOutput(
   const tables = mismatchingDependencyVersions
     .map((obj) => {
       const headers = [chalk.bold(obj.dependency), 'Usages', 'Packages'];
+
+      const usageCounts = obj.versions.map(
+        (versionObj) => versionObj.packages.length
+      );
+      const highestUsageCount = Math.max(...usageCounts);
+      const hasMultipleUsageCounts = !usageCounts.every(
+        (count) => count === highestUsageCount
+      );
+
       const rows = obj.versions
         .sort((a, b) => compareRanges(b.version, a.version))
         .map((versionObj) => {
+          const usageCount = versionObj.packages.length;
           const packages =
-            versionObj.packages.length > 3
+            usageCount > 3
               ? `${versionObj.packages.slice(0, 3).join(', ')}, and ${
-                  versionObj.packages.length - 3
-                } other${versionObj.packages.length - 3 === 1 ? '' : 's'}`
+                  usageCount - 3
+                } other${usageCount - 3 === 1 ? '' : 's'}`
               : versionObj.packages.join(', ');
           return [
             chalk.redBright(versionObj.version),
-            versionObj.packages.length,
+            // Bold the usage count if it's the highest, as long as it's not the only usage count present.
+            usageCount === highestUsageCount && hasMultipleUsageCounts
+              ? chalk.bold(usageCount)
```

Comment:
Is bold too subtle? Would it make sense to add a color as well?

Reply:
It's a bit subtle, but that could be okay because:
* It seems to be the general convention to bold the highest value in a column/list
* I don't really want to draw that much attention to it, as the highest dependency version is arguably more important than the highest usage count (in part because we will autofix to the highest dependency version) -- so I was a bit on the fence about this change to begin with
* I'm unaware of any other formatting (colors included) that makes sense